benchmarks/templates/benchmarks/base.html

```diff
@@ -94,10 +94,10 @@
     </div>
     <div id="navMenu" class="navbar-menu">
         <div class="navbar-end">
-            <a class="navbar-item" href="#leaderboard">Leaderboard</a>
-            <a class="navbar-item" href="#about">About</a>
-            <a class="navbar-item" href="#analysis">Compare</a>
-            <a class="navbar-item" href="#participate">Participate</a>
+            <a class="navbar-item" href="http://www.brain-score.org/#leaderboard">Leaderboard</a>
+            <a class="navbar-item" href="http://www.brain-score.org/#about">About</a>
+            <a class="navbar-item" href="http://www.brain-score.org/#analysis">Compare</a>
+            <a class="navbar-item" href="http://www.brain-score.org/#participate">Participate</a>
```

Comment:
IIRC in current `master`, these should resolve to the currently used URL which lets this run for development as well as in production

Reply:
Since there are multiple different pages using this navbar, I have it redirecting to the main page.